iothub/device/src/DeviceClient.cs

```diff
@@ -23,7 +23,7 @@ public class DeviceClient : IDisposable
         public const uint DefaultOperationTimeoutInMilliseconds = 4 * 60 * 1000;
 
         /// <summary>
-        /// The object serializer used by the <see cref="Client.PayloadConvention"/> set in <see cref="ClientOptions.PayloadConvention"/>. If no <see cref="Client.PayloadConvention"/> is set this will default to <see cref="NewtonsoftJsonPayloadSerializer"/>.
+        /// The object serializer used by the <see cref="Shared.PayloadConvention"/> set in <see cref="ClientOptions.PayloadConvention"/>. If no <see cref="Shared.PayloadConvention"/> is set this will default to <see cref="NewtonsoftJsonPayloadSerializer"/>.
```

Comment:
isn't "NewtonsoftJsonPayloadSerializer" a part of shared as well?

Reply:
Yes, this is because PayloadConvention is ambiguous because the class has a property named that as well.